### DISCUSSION TITLE
Show speed cameras along the route

### DISCUSSION THREAD

**Discussion by muralito** [organicmaps/organicmaps#5628] [Ideas]
Hi. I think it would be very useful (or to have an option to,) to show the speed or red light cameras along the calculated route, just like toll booths or toll roads.

**Comment by patepelo** [comment]
When planning?

I believe it's already working when navigating

**Reply by muralito** [reply]
Yes. I was not clear. Sorry.

BTW, I'm not sure the warning is working, but there is another thread with this issue.

And also, the search for "radar","speed camera" or "velocidad" nothing of these https://overpass-turbo.eu/s/1BWR are shown in the results. Maybe i'm not using the right search terms.
I would like to use the right query to check that the info is in the OM map.

**Reply by patepelo** [reply]
Would a user plan a route based on wether it has speed cameras?

**Comment by muralito** [comment]
> Would a user plan a route based on wether it has speed cameras?

Maybe not to the extent of changing the route, but it's very useful to be aware of the speed and red light cameras in advance.
Around here, it's not uncommon to have absurd speed limits, for example, in the middle of a rural area, the maximum speed changes abruptly from 90 to 45 (maybe for a small population or rural school) and they put a camera, most of the time with a desire for revenue collection and not for road safety.

I have the feeling that the speed warning is not working. I have tried to make the app warns me, of a 45 km/m limit, traveling at about 50 kn/h (speedometer) until approaching 50 mts to the camera, and i hear no sound, and see no warning. Most of the time i use the speed limiter, settled to the max speed of the road, so, strictly i'm not exceeding the speed, but if i know there is a camera, i would not accelerate a little and overtake a slower vehicle, and in if that case there is a camera, and i accelerate i would get be catched speeding.

**Comment by ftrebien** [comment]
Speed ​​limits are valid since the position of the sign. Any reduction in the speed limit must be announced in advance (as Waze does) so that drivers can reduce their speed in time. Prior warning about a speed limit reduction is obviously even more important where speed cameras are present, even if the driver is already driving within the reduced speed limit that applies only a little further ahead.
